news/2020-04-10-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "osu! Spotlights: New Team and League System"
+date: 2020-04-13 08:00:00 +0000
+---
+
+With our latest Spotlight season released very recently, we are underway reworking osu! Spotlights from ground up, starting with a brand new team and adding a whole new league system.
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving the curation and participation in osu! Spotlights a whole new twist.
+While we want to keep recommending beatmaps to you, we also want to add a whole new competitive and seasonal league closely attached to the recommended beatmaps. With this, you and your friends will have even more reason to delve deep into the best creations osu! has to offer.
```

Comment:
single line breaks don't do anything, if this was supposed to be a new paragraph u need another break